lib/services/ios-project-service.ts

```diff
@@ -388,27 +413,75 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 			await this.setupSigningForDevice(projectRoot, buildConfig, projectData);
 		}
 
-		if (buildConfig && buildConfig.codeSignIdentity) {
-			args.push(`CODE_SIGN_IDENTITY=${buildConfig.codeSignIdentity}`);
-		}
+		// These are now set in the .pbxproj, avoid passing them as arguments!
+		// if (buildConfig && buildConfig.codeSignIdentity) {
+		// 	args.push(`CODE_SIGN_IDENTITY=${buildConfig.codeSignIdentity}`);
+		// }
 
-		if (buildConfig && buildConfig.mobileProvisionIdentifier) {
-			args.push(`PROVISIONING_PROFILE=${buildConfig.mobileProvisionIdentifier}`);
-		}
+		// if (buildConfig && buildConfig.mobileProvisionIdentifier) {
+		// 	args.push(`PROVISIONING_PROFILE=${buildConfig.mobileProvisionIdentifier}`);
+		// }
 
-		if (buildConfig && buildConfig.teamId) {
-			args.push(`DEVELOPMENT_TEAM=${buildConfig.teamId}`);
-		}
+		// if (buildConfig && buildConfig.teamId) {
+		// 	args.push(`DEVELOPMENT_TEAM=${buildConfig.teamId}`);
+		// }
+
+		await this.xcodebuild(args, projectRoot, buildConfig.buildOutputStdio);
+		await this.createIpa(projectRoot, projectData, buildConfig);
+	}
 
-		// this.$logger.out("xcodebuild...");
-		await this.$childProcess.spawnFromEvent("xcodebuild",
-			args,
+	private async xcodebuild(args: string[], cwd: string, stdio: any = "inherit"): Promise<ISpawnResult> {
+		const localArgs = [...args];
+		const xcodeBuildVersion = await this.getXcodeVersion();
+		try {
+			if (helpers.versionCompare(xcodeBuildVersion, "9.0") >= 0) {
+				localArgs.push("-allowProvisioningUpdates");
+			}
+		} catch (e) {
+			console.warn("Failed to use xcodebuild with -allowProvisioningUpdates due to error: " + e);
+		}
+		if (this.$logger.getLevel() === "INFO") {
+			localArgs.push("-quiet");
+			this.$logger.info(`xcodebuild ${localArgs.join(" ")}`);
```

Comment:
Same. Now nothing is shown.
xcodebuild will not output a single line if run with -quiet and no errors or warning were present. So far the user had seen 10 screens of output in place of this with all the flags and environment variables passed to Xcode, various subtools that compile storyboards or copy PNGs. All of it is now collapsed to this line. I think this line is important otherwise the CLI will stall for several seconds and the user won't know that the native build actually started.

Again I can change the message if you provide me a better one, but I think it is important to appear in the default logging mode when -quiet is passed.